library-app/src/components/Books/BooksItem/BooksItem.tsx

```diff
@@ -1,14 +1,81 @@
+import { BookBodyDataGet } from '../../../models/bookData.model'
 import Card from '../../UI/Card'
-import book1 from '../../../assets/placeholderImg/book1.png'
 import styles from './BooksItem.module.css'
-const BooksItem = () => {
+import { useEffect, useState } from 'react'
+import imgPlaceholder from '../../../assets/placeholderImg/placeholder.jpeg'
+import Modal from '../../Layout/Modal'
+import { removeBookRequest } from '../../../services/BooksServices'
+import EditBookForm from '../BooksList/EditBookForm'
+
+export interface BookProps {
+  Book: BookBodyDataGet
+  isLoggedIn: boolean
+}
+
+const BooksItem = ({ Book, isLoggedIn }: BookProps) => {
+  const [isModalOpened, setIsModalOpened] = useState(false)
+
+  const [coverPlaceholder, setCoverPlaceholder] = useState('')
+
+  useEffect(() => {
+    setCoverPlaceholder(imgPlaceholder)
+  }, [])
+
   return (
     <Card>
       <div className={styles['book-holder']}>
-        <a href=''>
-          <img src={book1} alt='' className={styles['book-img']} />
-        </a>
+        <div className={styles['img-holder']}>
+          <img
+            src={Book.Cover ? `data:image/png;base64, ${Book.Cover}` : coverPlaceholder}
```

Comment:
why data:image/png;base64 ?

Reply:
the only way i can show img, bcs file is to big